File: app/src/main/java/fr/free/nrw/commons/review/ReviewEntity.java
package fr.free.nrw.commons.review;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

/**
 * Entity to store reviewed/skipped images identifier
 */
@Entity(tableName = "reviewed-images")
public class ReviewEntity {
    @PrimaryKey
    @NonNull
    String imageId;

    public ReviewEntity(String imageId) {
        this.imageId = imageId;
    }
}

File: app/src/main/java/fr/free/nrw/commons/review/ReviewEntity.kt
package fr.free.nrw.commons.review

import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * Entity to store reviewed/skipped images identifier
 */
@Entity(tableName = "reviewed-images")
data class ReviewEntity(
    @PrimaryKey
    val imageId: String
)
